apps/checkout-app/frontend/utils.ts

```diff
@@ -4,6 +4,7 @@ import { SettingsValues, UnknownSettingsValues } from "@/types/api";
 import { allSettingID, Item, NamedNode, Node, SettingID } from "@/types/common";
 import reduce from "lodash-es/reduce";
 import { CombinedError } from "urql";
+import { getSettingsPublicAccess } from "@/config/fields";
 
 export function parseJwt(token: string) {
```

Comment:
Let's get rid of this and use app id from app-bridge

Reply:
removed